statsd/options.go

```diff
@@ -293,3 +301,15 @@ func WithoutDevMode() Option {
 		return nil
 	}
 }
+
+// WithAgentMinVersion set the minimum version of the Agent receiving points
+// from the client. It allows better defaults depending on the version of the
+// Agent used. This will for example enable aggregation of histograms and
+// distributions when WithClientSideAggregation is enabled along side an Agent
+// >= 7.25.
+func WithAgentMinVersion(version string) Option {
```

Comment:
Typo with what ?

Reply:
Does `WithAgentMinVersion("7.25")` works as expected? The version can be misspelled  `WithAgentMinVersion("7.2.50")` instead of `WithAgentMinVersion("7.25.0")` or `WithAgentMinVersion("Agent 7.25.0")`

Reply:
I'm sorry, but I don't understand how having a type `Version` will help with any of those case.

We internally rely on semver parsing:
- misspelled version `"7.2.50"` vs `"7.25.0"` can't really be detected as both are valid versions
- `"7.25"` will be interpreted as `"7.25.0"`.
- `"Agent 7.25.0"` will return a ` Invalid Semantic Version` error.